third_party/color_picker/color-picker.coffee

```diff
@@ -0,0 +1,16 @@
+dom    = require('./dom')
```

Comment:
You must never include code without changing the copyright notice!!

If you must, you can create a git submodule, and then import the sources. But remove everything you took from quill from this repository!